cmd/helm/package.go

```diff
@@ -75,6 +76,10 @@ func newPackageCmd(out io.Writer) *cobra.Command {
 				if err != nil {
 					return err
 				}
+				if _, err := os.Stat(path); os.IsNotExist(err) {
```

Comment:
We can bubble out any `Stat` errors we get. Let's not restrict it to just does-no-exist errors.

I just tried this -

```
$ mkdir yet-another-dummy
$ sudo chmod 000 yet-another-dummy
$ ls -al | rg dummy
d---------    3 karuppiahn  staff       96 Jan 19 12:41 yet-another-dummy
$ helm package yet-another-dummy/this-does-not-exist
Error: validation: chart.metadata is required
```

Which leads us to the same issue as before, so an error message that's easier for the user to debug could be just bubble up what error `Stat` gives, which in this case looks like 

```
$ helm package yet-another-dummy/this-does-not-exist
Error: stat /Users/karuppiahn/yet-another-dummy/this-does-not-exist: permission denied
```

Reply:
@anshulvermapatel ^

Reply:
One way to avoid such issues is - write as little code as possible to pass the test. In this case, I think this is enough, what do you think?

```
if _, err := os.Stat(args[i]); err != nil {
    return err
}
```

Reply:
And then if you have other cases in mind, you can write tests for it and write more code. For example, other kind of errors like permission denied and so on. 😄